app/blueprints/self_serve/routes.py

```diff
@@ -128,12 +98,31 @@ def view_form_questions():
     html = print_html(print_data)
     return render_template("view_questions.html", section_name=form_config["title"], question_html=html)
 
+@self_serve_bp.route("/section_questions", methods=["POST"])
+def view_section_questions():
+    # form_config = generate_form_config_from_request()
```

Comment:
shall we remove these commented lines or we'll add TODO to identify the things that we need to do if we need these commented lines